### PR TITLE
L-733 Rebrand Logtail into BetterStack.Logs.NLog

### PR DESCRIPTION
**Pull request by PetrHeinz**
I'm working on detaching NLog dependency from our .NET client (into `BetterStackHQ/logs-client-serilog`), which is a good moment to move our clients out of `logtail` organization and out of beta.

Main is currently just a copy of https://github.com/logtail/logtail-dotnet. I've squashed the history to avoid linking historical commits with PR numbers to random newly created PRs.

After merge:
- [x] Create a tag and release on Github
- [x] Release `BetterStack.Logs.NLog` as `1.0.0` on nuget.org
- [x] Test example project with released nuget
- [x] Update docs
- [x] Update Notion instructions for releasing
- [x] Mark original repo as deprecated + link here

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
-# [Better Stack](https://betterstack.com/logs) .NET client
+# [Better Stack](https://betterstack.com/logs) .NET NLog client
 
-📣 Logtail is now part of Better Stack. [Learn more ⇗](https://betterstack.com/press/introducing-better-stack/)
-
-[![Better Stack dashboard](https://github.com/logtail/logtail-python/assets/10132717/e2a1196b-7924-4abc-9b85-055e17b5d499)](https://betterstack.com/logs)
+[![Better Stack dashboard](https://raw.githubusercontent.com/BetterStackHQ/logs-client-nlog/main/dashboard.png)](https://betterstack.com/logs)
```
Comment (PetrHeinz) [inline README.md:3]: Image will start working with merge.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
 
 [![ISC License](https://img.shields.io/badge/license-ISC-ff69b4.svg)](LICENSE.md)
-[![Nuget version](https://badge.fury.io/nu/Logtail.svg)](https://www.nuget.org/packages/Logtail)
+[![Nuget version](https://badge.fury.io/nu/BetterStack.Logs.NLog.svg)](https://www.nuget.org/packages/BetterStack.Logs.NLog)
```
Comment (PetrHeinz) [inline README.md:6]: Badge will start working with release on nuget.

File: example-project/README.md [hunk 3]
```diff
@@ -50,16 +52,16 @@ In the root directory of the project create the `nlog.config` file or copy the f
...
         internalLogFile="internal.txt">
 	<extensions>
-		<add assembly="Logtail" />
+		<add assembly="BetterStack.Logs.NLog" />
 	</extensions>
 
 	<targets>
 		<!-- Dont forget to change SOURCE_TOKEN to your actual source token-->
-		<target xsi:type="Logtail" name="logtail" layout="${message}" sourceToken="SOURCE_TOKEN" />
+		<target xsi:type="BetterStack.Logs" name="mybetterstack" layout="${message}" sourceToken="SOURCE_TOKEN" />
```
Comment (curusarn) [inline example-project/README.md:60]: Nit: I would prefer just `betterstack` over `mybetterstack`.

**Review by curusarn — APPROVED** [review]
Thanks a lot Petr!

Left one nitpick.

File: example-project/README.md (hunk 3 above)
Comment (PetrHeinz) [reply example-project/README.md:60]: I wasn't sure myself, but wanted to make sure people would understand this is a string they have control over 🤷

Comment (curusarn) [reply example-project/README.md:60]: That also makes sense - feel free to keep the name whichever way you prefer. 👍

**Pull request merged**